src/commands/distribute/releases/edit.ts

```diff
@@ -70,23 +78,55 @@ export default class EditReleaseCommand extends AppCommand {
       handleHttpError(error, false, "failed to load release details");
     }
 
+    let enabled: boolean = null
+    if(this.state === "enabled" || this.state === "disabled"){
```

Comment:
Since you validated the parameters, the condition could simply be `!_.isNil(this.state)`.

Reply:
If the parameter is required then we don't need this check at all.